docs/resources/azurerm_key_vault.md.erb

```diff
@@ -0,0 +1,109 @@
+---
+title: About the azurerm_key_vault Resource
+platform: azure
+---
+
+# azurerm\_key\_vault
+
+Use the `azurerm_key_vault` InSpec audit resource to test properties and configuration of
+an Azure Key Vault.
+<br />
```

Comment:
I've been adding a link in this section to the REST API documentation in Azure. Depending on when things merge you may want to do the same here. Here's an example of what I've been putting:

```
## Azure REST API version

This resource interacts with version `VERSION` of the Azure Graph API. For more
information see the [official Azure documentation](URL).

At the moment, there doesn't appear to be a way to select the version of the
Azure API docs. If you notice a newer version being referenced in the official
documentation please open an issue or submit a pull request using the updated
version.
```

Reply:
Good call. I can Add this to the SQL resources too.